tools/devops/xcode-deps.csx.in

```diff
@@ -0,0 +1,22 @@
+#r "_provisionator/provisionator.dll"
+
+using System.IO;
+using System.Linq;
+
+using static Xamarin.Provisioning.ProvisioningScript;
+
+Xcode ("@XCODE_VERSION@").XcodeSelect (allowUntrusted: true);
+
+// provisionator knows how to deal with this items
+Item ("@MONO_PACKAGE@", kind: ItemDependencyKind.AtLeast);
```

Comment:
I'm not sure the `AtLeast` part is right, I think we currently install this exact package, which means we'll want the exact version.